### ISSUE TITLE
bug: Google Signin flow breaks with firebase 10+ on PWA, works after downgrade

### ISSUE THREAD

**Issue by woutersteven** [capawesome-team/capacitor-firebase#412]
### Plugin(s)

- [ ] Analytics
- [ ] App
- [ ] App Check
- [X] Authentication
- [ ] Crashlytics
- [ ] Cloud Messaging
- [ ] Performance
- [ ] Remote Config

### Did you test the latest version?

- [X] I use the latest version

### Platform(s)

- [ ] Android
- [ ] iOS
- [X] Web

### Current behavior

Using the latest version of the plug-in with firebase 10.1.0 compiles, but the authentication flows break in a PWA setup and using signInWithGoogle. The error message in the browser is: "requested action is invalid".

It affects the PWA version (not Android and not tested on iOS) and downgrading the firebase package to v9.23 resolves the issue.

### Expected behavior

The redirect is expected to function normally, since it is marked as compatible.

### Reproduction

None available

### Steps to reproduce

It's probably a matter of upgrading the included JS SDK or some other minimal change

### Capacitor doctor

Capacitor Doctor   

Latest Dependencies:

  @capacitor/cli: 5.2.2
  @capacitor/core: 5.2.2
  @capacitor/android: 5.2.2
  @capacitor/ios: 5.2.2

Installed Dependencies:

  @capacitor/core: 5.2.2
  @capacitor/cli: 5.2.2
  @capacitor/ios: 5.2.2
  @capacitor/android: 5.2.2

### Before submitting

- [X] I understand that incomplete issues (e.g. without reproduction) are closed.

**Comment by robingenz** [comment]
I am currently using `@capacitor-firebase/authentication@5.1.0` with `firebase@10.1.0` in another project and the Google Sign-In on web works without any problems so far.

**Comment by robingenz** [comment]
Have you already tried to create a Minimal, Reproducible Example? Then you should be able to find out when it stops working. I myself only use `@capacitor-firebase/authentication` and `firebase` but no AngularFire. I don't think it's because of that but try to start with as few dependencies as possible and a blank app.

**Comment by tommyc38** [comment]
@woutersteven2020 I was hoping you might be able to help me.  I have an ionic/angular/angularfire/capacitor app that is bare bones with a few routes and and an auth service.  I have implemented google auth via this library and it logs me in without any popup even though I listed scopes and explicitly set mode to 'popup' like you have above.  Any ideas on what I am missing?  The authentication works, there is just no google interface presented.  I am talking Android specifically here.  The web works as expected.  I feel like I am missing some kind of config somewhere.  I would really appreciate any help!
